allegro.h

```diff
@@ -682,7 +688,9 @@ typedef class Alg_track : public Alg_event_list {
     Alg_track(Alg_event_list_ref event_list, Alg_time_map_ptr map, 
               bool units_are_seconds);
     virtual ~Alg_track() { // note: do not call set_time_map(NULL)!
-        if (time_map) time_map->dereference(); time_map = NULL; }
+        if (time_map) { time_map->dereference(); }
```

Comment:
Agreed the original is not very consistent with format -- maybe I was trying to see how short, compact code fairs in small simple cases. If you want to be stricter, put newline after { and before }.